src/containers/mainPage/iPhoneComponent/header/burgerMenu/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { MdSearch } from 'react-icons/md';
+import { NavLink } from 'react-router-dom';
+import styles from './styles.module.scss';
+
+class Burger extends Component {
+  state = {
+    menuIsActive: false,
+  };
+
+  burgerHandle = () => {
+    this.setState(state => ({ menuIsActive: !state.menuIsActive }));
+  };
+
+  render() {
+    const { menuIsActive } = this.state;
+    const { log, reg } = this.props;
```

Comment:
This component has extends from the header, so I can't resolve this situation now, because we can rewrite code in the header component for resolving this.